ignition/examples.md

```diff
@@ -40,10 +40,12 @@ WantedBy=multi-user.target
 
 In many scenarios, it may be preferable to use btrfs for the root filesystem.
```

Comment:
Although CoreOS uses ext4 by default, the btrfs filesystem may be an appropriate choice for the system root partition in some scenarios.